### PR TITLE
Gujarati phonetic layout

### PR DESCRIPTION
**Pull request by Yogesh-B**
Added Gujarati phonetic layout.

- [x] added layout xml
- [x] added changes in `KeyModifier.java` for shift key combinations
- [x] re-generated `values/layouts.xml` 
- [x] re-generated `check_layout.output`

Tested debug APK build(gradle and java), working fine on my device.

### CODE REVIEW

File: srcs/layouts/guj_phonetic_in.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<keyboard name="ગુજરાતી ફોનેટિક - Gujarati Phonetic" script="gujarati">
+  <row>
```
Comment (Julow) [inline srcs/layouts/guj_phonetic_in.xml:3]: The top row doesn't look good in combination to the built-in number row (that can be enabled in the settings). Do you think every users of this layout will want to have this row ?

Also, the layout is very high due to top row. It takes almost half of the screen on my device. You can make the top row a bit smaller with `height="0.75"` (like the built-in number row).

File: srcs/layouts/guj_phonetic_in.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    <key key0="1" key1="`" key2="૧" key3="!" key4="esc"/>
+    <key key0="2" key1="~" key2="૨" key3="\@" key4="૱"/>
```
Comment (Julow) [inline srcs/layouts/guj_phonetic_in.xml:5]: The layout contains a lot of ASCII symbols like `~`, `@`, `^`, etc.. Are these really used ?

**Review by Julow — APPROVED** [review]
Looks good :) I have left some comments but I'd be happy to merge it like this if you think it's ready.

File: srcs/layouts/guj_phonetic_in.xml (hunk 2 above)
Comment (Yogesh-B) [reply srcs/layouts/guj_phonetic_in.xml:5]: mostly not much useful frequently but kept it as default layout has it and most keyboards contains these.

File: srcs/layouts/guj_phonetic_in.xml (hunk 1 above)
Comment (Yogesh-B) [reply srcs/layouts/guj_phonetic_in.xml:3]: we can do something like this to remove the top row. [**as you see in the Screen Shot, numbers are shown properly in keyboard but output is not in devanagari. I feel like there is a bug. It devanagari numbers work fine in normal view, just this number view is not working proper]

File: srcs/layouts/guj_phonetic_in.xml (hunk 2 above)
Comment (Julow) [reply srcs/layouts/guj_phonetic_in.xml:5]: Makes sense.

File: srcs/layouts/guj_phonetic_in.xml (hunk 1 above)
Comment (Julow) [reply srcs/layouts/guj_phonetic_in.xml:3]: The devanagari digits are specified here: https://github.com/Julow/Unexpected-Keyboard/blob/master/srcs/juloo.keyboard2/KeyModifier.java#L1060
Are they wrong ?

To change the numpad digits when your layout is visible, you'd need to add a new similar map and call it from here: https://github.com/Julow/Unexpected-Keyboard/blob/master/srcs/juloo.keyboard2/KeyModifier.java#L100

Comment (Yogesh-B) [reply srcs/layouts/guj_phonetic_in.xml:3]: The first screen shot just shows the layout in Devanagari(or any other language), but when typing, it puts output as english numbers. I think it is using values from https://github.com/Julow/Unexpected-Keyboard/blob/master/res/xml/numpad.xml .  

I tried adding Gujarati number in the java file you suggested, but had same issue as mentioned above.

**Comment by Yogesh-B** [comment]
I could not find option to enable script specific numeric row, hence currently adding first row permanent and have made changes to row height so that it does not take up space.

Now the layout looks quite good and feels almost same as experience on PC. No issue in merging it.

Numeric layout related issue can be done in another pull/issue.

File: srcs/layouts/guj_phonetic_in.xml (hunk 1 above)
Comment (Julow) [reply srcs/layouts/guj_phonetic_in.xml]: You are right, there's a serious bug in the numpad. I fixed it in 8c7559d and fixed the number row in 8b5d971.

You can now remove the number row from your layout and enable the number row in the settings.

Note that the layout must also contain the numerals for people that don't use the number row option.

**Comment by Julow** [comment]
Now that the number row and numpad are fixed on master, could you pull master and update the layout ?

I've added support for the gujarati numerals in this branch: https://github.com/Julow/Unexpected-Keyboard/tree/gujarati_numpad_script
Though, I'm not sure I got them right so I'd prefer if you could push that to your branch or push the implementation you had earlier.

File: srcs/layouts/guj_phonetic_in.xml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<keyboard name="ગુજરાતી ફોનેટિક - Gujarati Phonetic" script="gujarati">
+  <row>
+    <key key0="ટ" key1="`" key2="1" key3="!" key4="esc"/>
```
Comment (Julow) [inline srcs/layouts/guj_phonetic_in.xml:4]: The western numerals are on the keyboard instead of the gujarati ones. Is this intended ?

**Review by Julow — APPROVED** [review]
The layout looks nice :) I have a comment but I would merge before next release anyway.

Many thanks for helping improve the keyboard :)

**Pull request merged**